Specific Integrations to point to our page rather than official page

SmritiSatyanV opened the pull request:
Updated index.html
Signed-off-by: SmritiSatyanV <smriti@union.ai>

SmritiSatyanV committed:
Updated index.html

Signed-off-by: SmritiSatyanV <smriti@union.ai>

samhita-alla commented on index.html:
```diff
@@ -1848,7 +1848,7 @@ <h2 class="text-white text-center text-3xl font-bold pb-4">
         style="align-content: center; display: inherit"
       >
-        <a href="https://horovod.ai" target="_blank">
+        <a href="https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/kubernetes/kfmpi/index.html" target="_blank">
```
Can you link the text to https://docs.flyte.org/projects/cookbook/en/latest/auto/case_studies/ml_training/spark_horovod/index.html?

samhita-alla commented on index.html:
```diff
@@ -1875,7 +1875,7 @@ <h2 class="text-white text-center text-3xl font-bold pb-4">
         style="align-content: center; display: inherit"
       >
-        <a href="https://www.snowflake.com/" target="_blank">
+        <a href="https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/external_services/snowflake/#" target="_blank">
```
```suggestion
        <a href="https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/external_services/snowflake/index.html" target="_blank">
```

SmritiSatyanV committed:
Added Modin integration to flyte.org

Added image
Updated index and gemfile
Signed-off-by: SmritiSatyanV <smriti@union.ai>

SmritiSatyanV committed:
Changed png image to svg

Signed-off-by: SmritiSatyanV <smriti@union.ai>

SmritiSatyanV committed:
Removed svg and added png

Signed-off-by: SmritiSatyanV <smriti@union.ai>

cosmicBboy commented:
Ah, sorry @samhita-alla I think this conflicts with this PR https://github.com/flyteorg/flyteorg.github.io/pull/100.

Feel free to revert the changes I made there, it's better to point to the docs pages via the logo links

SmritiSatyanV committed:
Updated index.html

Fixed render
Signed-off-by: SmritiSatyanV <smriti@union.ai>

SmritiSatyanV committed:
Merge branch 'development' into Integration-redirect

SmritiSatyanV committed:
Updated index.html

Changes due to https://github.com/flyteorg/flyteorg.github.io/pull/100 were reverted
Signed-off-by: SmritiSatyanV <smriti@union.ai>

samhita-alla committed:
resolution fixes

Signed-off-by: Samhita Alla <aallasamhita@gmail.com>
